.github/workflows/edocs-dynamic-sync.yml

```diff
@@ -1,13 +1,5 @@
-name: Synchronize with eDocs
-on:
-  workflow_dispatch:
-    inputs:
-      content:
-        type: choice
-        description: Content to be synchronized
-        options: 
-        - static
-        - all
+name: Synchronize Dynamic docs with EPDocs
```

Comment:
I would like to group the workflows in the left menu which is alphabetically order:

![image](https://user-images.githubusercontent.com/4663897/202442636-08e18b0d-69ba-4b9f-b8fb-72f6a2fe1001.png)

For that  I would change the name to:

```suggestion
name: EPDocs - Dynamic Content PR creation
```